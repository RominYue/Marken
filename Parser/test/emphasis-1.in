*single asterisks*
_single underscores_
**double asterisks**
__double underscores__
un*frigging*believable
\*this text is surrounded by literal asterisks\*