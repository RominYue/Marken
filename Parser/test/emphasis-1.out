<p><em>single asterisks</em>
<em>single underscores</em>
<strong>double asterisks</strong>
<strong>double underscores</strong>
un<em>frigging</em>believable
*this text is surrounded by literal asterisks*</p>